presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -103,8 +103,8 @@ public class FeaturesConfig
     private String historyBasedOptimizerPlanCanonicalizationStrategies = "IGNORE_SAFE_CONSTANTS";
     private boolean logPlansUsedInHistoryBasedOptimizer;
     private boolean enforceTimeoutForHBOQueryRegistration;
-    private boolean redistributeWrites = true;
-    private boolean scaleWriters;
+    private boolean redistributeWrites;
+    private boolean scaleWriters = true;
```

Comment:
Does this mean that Sapphire jobs will need explicitly setting this session property to false? https://github.com/prestodb/presto/blob/ec60c1377a0a2ff4b2f0ebce6d831b7eca71fddf/presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkSettingsRequirements.java#L55

Reply:
Presto on Spark jobs already do: 
https://github.com/prestodb/presto/blob/ec60c1377a0a2ff4b2f0ebce6d831b7eca71fddf/presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkSettingsRequirements.java#L93